### DISCUSSION TITLE
website down?

### DISCUSSION THREAD

**Discussion by kirk-hail** [keplergl/kepler.gl#2550] [General]
Looks like the website is down; adjust chrome to allow to view anyway and it gets this:
<img width="1024" alt="Screenshot 2024-03-27 at 9 26 24 AM" src="https://github.com/keplergl/kepler.gl/assets/157629987/41f9a019-9816-461f-8fd3-f4ab6a3a4955">

**Comment by chrisgervang** [comment]
The website works when I test it. It might have been a temporary issue? https://kepler.gl
